CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ project(
         "The 'build-all' instructions for all examples for the book CMake Best Practices"
     LANGUAGES CXX
 )
+include(CTest)
 enable_testing()
```

Comment:
include(CTest) makes enable_testing redundant,

Quoting from CMake documentation for enable_testing() (https://cmake.org/cmake/help/latest/command/enable_testing.html):

```
... This command is automatically invoked when the CTest module is included, except if the BUILD_TESTING option is turned off. ...
```

Reply:
True, will remove `enable_testing()`